eng/common/TestResources/New-TestResources.ps1

```diff
@@ -527,6 +527,8 @@ try {
     if ($CI -and $Environment -eq 'AzureCloud' -and $env:PoolSubnet) {
         $templateParameters.Add('azsdkPipelineSubnetList', @($env:PoolSubnet))
     }
+    # Some arm/bicep templates may want to change deployment settings (e.g. local auth) in sandboxed TME tenants
+    $templateParameters.Add('tenantIsTME', ($context.Tenant.Name -like '*TME*'))
```

Comment:
@deyaaeldeen, this looks good - will this just get merged into azure-sdk-tools at some point?

Reply:
@benbp when is the ETA for the merge of your PR?